.circleci/config.yml

```diff
@@ -50,8 +50,9 @@ jobs:
       - run:
           name: Start Runner
           command: |
-            sudo launchctl bootstrap user/`stat -f %u` ~/Library/LaunchAgents/com.circleci.runner.plist
-            launchctl load ~/Library/LaunchAgents/com.circleci.runner.plist
+            sudo mv ~/Library/LaunchAgents/com.circleci.runner.plist /Library/LaunchAgents/
+            launchctl bootstrap user/$(id -u) /Library/LaunchAgents/com.circleci.runner.plist
+            launchctl print user/$(id -u)/com.circleci.runner
```

Comment:
Updated the CI smoke test to match the new instructions: https://app.circleci.com/pipelines/github/CircleCI-Public/homebrew-circleci/352/workflows/454e6929-ac78-4991-b917-8b10d518e2ad/jobs/350/parallel-runs/0/steps/0-104